spec/models/assignment_spec.rb

```diff
@@ -1035,7 +1100,7 @@
   describe '#grade_distribution_as_percentage' do
     before :each do
       @assignment = create(:assignment)
-      5.times { create(:rubric_criterion, assignment: @assignment) }
+      5.times { create(:rubric_criterion, assignment: @assignment, peer_visible: true) }
```

Comment:
Calculating Assignment grades should not require `peer_visible`criteria to be successful. Does this method `grade_distribution_as_percentage` need to be updated as well?